websites/D/DscJobs/presence.ts

```diff
@@ -0,0 +1,131 @@
+const presence = new Presence({
+    clientId: "846309777508007946"
+  }),
+  browsingStamp = Math.floor(Date.now() / 1000),
+  [page] = document.location.href.split("page=")[1].split("&"),
+  [staffPage] = document.location.href.split("page=")[1],
+  filters = document.location.href.includes("&"),
+  [search] = document.location.href.split("term=")[1].split("&");
```

Comment:
Allow me to introduce you to URLSearchParams

Reply:
This please